src/main/java/com/example/pokebattlez/battle/model/Type.java

```diff
@@ -0,0 +1,7 @@
+package com.example.pokebattlez.battle.model;
+
+public enum Type {
```

Comment:
Type of what?

Reply:
Both Pokemon types as well as Move types. They have the same possible types. 
If you are interested: https://bulbapedia.bulbagarden.net/wiki/Type